sssd_test_framework/roles/client.py

```diff
@@ -70,6 +71,11 @@ def __init__(self, *args, **kwargs) -> None:
         Managing local overrides users and groups.
         """
 
+        self.pam: PAMUtils = PAMUtils(self.host, self.fs)
```

Comment:
It does not have to be here, it is inherited from BaseLinuxRole

Reply:
I think Pavel's comment is still valid so I'm reopening this conversation

Reply:
Yes, I this is an artifact of fighting with my IDE, removed again.